catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableImpl.java

```diff
@@ -95,15 +91,45 @@ public CatalogMigratableImpl(@NotNull String description, @NotNull CatalogFramew
         this.filterBuilder = filterBuilder;
         this.fileWriter = fileWriter;
         this.config = config;
+
+        this.taskManager = new MigrationTaskManager(config, fileWriter);
```

Comment:
Should be able to simplify this constructor by delegating to the other one. See my comment on the `MigrationTaskManager` constructor below.